{"name": "test_login_with_wrong_credentials", "status": "passed", "steps": [{"name": "Click on Login Button", "status": "passed", "start": 1623200401133, "stop": 1623200401133}, {"name": "Entered email id", "status": "passed", "start": 1623200401686, "stop": 1623200401686}, {"name": "Entered password", "status": "passed", "start": 1623200403156, "stop": 1623200403156}, {"name": "Wrong credentials is displayed", "status": "passed", "start": 1623200403675, "stop": 1623200403675}], "attachments": [{"name": "log", "source": "5b85f025-c7eb-49d0-adc8-aaac649c3e9c-attachment.txt", "type": "text/plain"}], "start": 1623200389096, "stop": 1623200403675, "uuid": "02515997-9c65-46d2-b740-53d18371d709", "historyId": "fec9b1445d3106ed0be16933b6a0fa4a", "testCaseId": "f40b97252b1beb284aec8fcfca5b6025", "fullName": "TestSuite.LoginTest#test_login_with_wrong_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "tag", "value": "run(order=2)"}, {"name": "suite", "value": "TestSuite"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "vadim-asus"}, {"name": "thread", "value": "90621-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSuite"}]}